changelog.d/gh-7772.fixed

```diff
@@ -0,0 +1 @@
+Enable automatic removal of matched codes by allowing an empty string in the fix field.
```

Comment:
Usually we name this after the GitHub issue that we're fixing, not the pull request that fixed it

Reply:
Fixed. I believed that the ID was a pull request number because I searched for an existing file `gh-7757` as an example, and there is a PR 7757. Thank you for the clarification.